packages/yoga/src/Avatar/native/Avatar.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { BuildingFilled } from '@gympass/yoga-icons';
+import { string, func, number } from 'prop-types';
+import Box from '../../Box';
+import Image from './Image';
+import Placeholder from './Placeholder';
+
+const Avatar = ({
+  src,
+  fill,
+  placeholder,
+  width = 48,
+  height = 48,
+  borderRadius = 'small',
+  type = 'default',
+}) => {
+  const icon = placeholder || BuildingFilled;
+  return (
+    <Box
+      bgColor="elements.selectionAndIcons"
+      display="flex"
+      width={width}
+      height={height}
+      alignItems="center"
+      justifyContent="center"
+      elevation="small"
+      borderRadius={borderRadius}
+    >
+      {src ? (
+        <Image type={type} source={src} />
+      ) : (
+        <Placeholder fill={fill} icon={icon} />
+      )}
+    </Box>
+  );
+};
+
+Avatar.propTypes = {
+  src: string,
+  placeholder: func,
+  fill: string,
+  width: number,
+  height: number,
+  borderRadius: string,
+  type: string,
+};
+
+Avatar.defaultProps = {
+  src: undefined,
+  placeholder: undefined,
+  fill: undefined,
+  width: undefined,
+  height: undefined,
+  borderRadius: undefined,
+  type: undefined,
+};
```

Comment:
Do we need to set the defaultProps? The `Avatar` parameters already have default values.
I think we should use one or the other. I prefer the function parameters as they end up helping in Intelisense.

```js
const Avatar = ({
  src,
  fill,
  placeholder,
  width = 48,
  height = 48,
  borderRadius = 'small',
  type = 'default',
})
````

OR

```js
Avatar.defaultProps = {
  src: undefined,
  placeholder: undefined,
  fill: undefined,
  width: 48,
  height: 48,
  borderRadius: 'small',
  type: 'default',
};
```

Reply:
@leonardokl  I prefer too using in parameters, but the lint of project required have defaultProps.

Reply:
@leonardokl Done! I added in default props only this values